src/App.tsx

```
@@ -4,8 +4,9 @@ import MainScreen from '@_pages/MainScreen';
 import MyInfoPage from '@_pages/MyinfoPage';
 import UserAlertListPage from '@_pages/UserAlertListPage';
 import SosHistoryPage from '@_pages/SosHistoryPage';
+import SosUserDetailPage from '@_pages/SosUserDetailPage';
```

ESLint

[error] `@_pages/SosUserDetailPage` import should occur after import of `@_pages/SosTimelinePage`

(import/order)